File: js/pigLatin.js
exports.translate = function(word) {
    let vowels = ['a', 'e', 'i', 'o', 'u']
    pigLatin = []

    let splitWords = word.split(' ');
    let count = 0
    for(i=0; i<splitWords.length; i++) {
        for(j=0; j<splitWords[i].length; j++) {
            if(vowels.includes(splitWords[i][j]) == false) {
                if(splitWords[i][j] == 'q' && splitWords[i][j+1] == 'u') {
                    splitWords[i] += (splitWords[i][j] + splitWords[i][j+1])
                    count+=2
                } else {
                    splitWords[i] += splitWords[i][j];
                    count++
                }
            } else {
                break;
            }
        }
        
        let temp = splitWords[i].substring(count) + 'ay'
        pigLatin.push(temp)
        count = 0
    }
    return pigLatin.join(' ')
};

// console.log(exports.translate('quiet'))

File: python/pig_latin.py
def translate(word_or_phrase):
  vowels = ['a', 'e', 'i', 'o', 'u']
  pig_latin = []

  split_words = word_or_phrase.split(' ')
  count = 0
  for i in range(0, len(split_words)):
    for j in range(0, len(split_words[i])):
      if split_words[i][j] not in vowels:
        if split_words[i][j] == 'q' and split_words[i][j+1] == 'u':
          split_words[i] += (split_words[i][j] + split_words[i][j+1])
          count+=2
        else:
          split_words[i] += split_words[i][j]
          count+=1
      else:
        break
    
    temp = split_words[i][count:] + 'ay'
    pig_latin.append(temp)
    count = 0

  return " ".join(pig_latin)